bin/generate-commit-log

```diff
@@ -11,6 +11,8 @@ IGNORE_COMMITS = [
     "^chore",
     "^pontoon",
     "UTF.*Pontoon",
+    "^Update .* to .*",
```

Comment:
This regex is too permissive.  But there's a better way.  When the script go through the log messages, it builds a dictionary for each.  So `message.get("from")` will give you the author of the commit, and you could match that against renovate.